README.md

```diff
@@ -11,10 +11,17 @@ gen3 integration tests - run by https://jenkins.planx-pla.net/ via a `Jenkinsfil
 docker run -d -p 4444:4444 --name=selenium --rm -v /dev/shm:/dev/shm selenium/standalone-chrome
 ```
 
-Edit your `/etc/hosts` file to point the `selenium-hub` host to your Docker container running on `localhost`.
+```
+# start influxdb (test run metrics are captured here, optional but you will see some errors)
```

Comment:
Oh, this instruction might be interpreted as if both local DNS tweaks were optional but only the influxdb one is.
The first one (`selenium-hub`) is actually mandatory.

But we can rephrase this later anyway. 👍